docs/Advanced-Usage/Workloads/Defining-Custom-Workloads.rst

```diff
@@ -41,8 +41,8 @@ inline.
 
 
 The following examples use the default buildroot-based linux
-distribution (br-base). In order to customize Fedora, you refer to the :ref:`booting-fedora`
-page. Imporantly, Fedora currently does not support the "command" option for
+distribution (br-base). In order to customize Fedora, you should refer to the :ref:`booting-fedora`
+page. Importantly, Fedora currently does not support the "command" option for
```

Comment:
This is not true anymore. Command works fine, you may as well remove this line if you're messing with the file anyway.

Reply:
Removed it, thanks.